platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/migration/SystemConfigurationMigrationTest.java

```diff
@@ -516,6 +557,14 @@ private Path createTempDdf() throws IOException {
         Files.createFile(rootTempDir.resolve(INVALID_KEYSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_TRUSTSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_CRL_REL_PATH));
+
+        existingFilePath = Paths.get(ddfHome.resolve("orig.config")
+                .toString());
+        symLinkPath = Paths.get(ddfHome.resolve("symlink.config")
```

Comment:
Don't think we need to call `Paths.get()` here, `ddfHome.resolve()` should already return `Path` object.